hypershift-operator/controllers/nodepool/testdata/zz_fixture_TestAPIServerHAProxyConfig_When_noproxy_matches_internalAddress_it_should_create_an_haproxy.yaml

```diff
@@ -0,0 +1 @@
+zz_fixture_TestAPIServerHAProxyConfig.yaml
```

Comment:
Something went wrong here, should be serializing the config as YAML

Reply:
they are syslinks and I think that's how GH represent the change

Reply:
What are they linked to? Did they get generated from `UPDATE=true go test ./...` ? I never saw symlinked fixtures before

Reply:
nvm symlink was causing troubles so I recreated then with test